frontend/package.json

```diff
@@ -19,6 +19,7 @@
   "homepage": "https://github.com/bcgov/nr-get-token",
   "dependencies": {
     "axios": "^0.19.0",
+    "axios-mock-adapter": "^1.17.0",
```

Comment:
This should be a dev dependency - it is not needed to run the frontend

Reply:
npm i axios-mock-adapter -D